sap-xmake-build.sh

```diff
@@ -0,0 +1,119 @@
+#!/bin/bash -l
+
+# This build script is only applicable to Spark without Hadoop and Hive
+
+PACKAGE_BRANCH="alti-branch-0.5"
+HADOOP_VERSION="2.7.4"
```

Comment:
I think we are now at `2.7.7`? Do we want to promote this now or later? I know we are inheriting existing env settings. Wondering when is a good time to bump this up?